content/applications/inventory_and_mrp/manufacturing/advanced_configuration/sub_assemblies.rst

```diff
@@ -12,7 +13,29 @@ manufacturing by defining each procurement and production step separately.
 Sublevel |BOMs| (subassemblies or semifinished products) streamline these production workflows, and
 are beneficial when the subassembly is used across multiple finished products (meaning they'd appear
 on multiple top-level |BOMs|). The more complicated a product is to build or procure, the more value
-a multilevel |BOM| can provide.
+a multilevel |BOM| can provide. Because of this, component and subassembly replenishment planning is
+vital to ensuring multilevel |BOMs| run smoothly.
+
+Why replenishment planning is important
+=======================================
+
+:doc:`Replenishment <../../inventory/warehouses_storage/replenishment>` is critical for multilevel
+BoMs to prevent bottlenecks, manage lead times, and optimize inventory. Without it, missing
+components can halt production, delay orders, and increase costs. A well-planned replenishment
+strategy ensures
+
+
+
+:doc:`just-in-time availability <../../inventory/warehouses_storage/replenishment/reordering_rules>`
+, :doc:`automates procurement <../../inventory/warehouses_storage/replenishment/reordering_rules>`,
+balances stock levels, and keeps supply chains efficient. This minimizes delays, reduces manual
+effort, and ensures smooth manufacturing.
+
+Manufacturing orders (MO)s that come from a |BOM| require all components to be available before the
+|MO| can be completed. The :guilabel:`Component Status` for that |MO| provides this information.
+Learn how to :doc:`check an MO's component status <../basic_setup/bill_configuration>`
+
+.. _Create a Multilevel BoM:
```

Comment:
Custom anchors must not contain spaces and are to follow [a convention](https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#custom-anchors)
```suggestion
.. _manufactuing/advanced/create-multilevel-bom:
```